.github/containers/ruby/spack.yaml

```diff
@@ -0,0 +1,80 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - python
+  - py-torch
+  - py-scipy
+  - py-torchvision
+  - py-horovod
+  - py-pillow
+  - py-h5py
+  - py-pandas
+  - py-scikit-learn
+  - py-matplotlib
+  - py-pip
+  - py-kosh
+  - py-mpi4py
+  - mfem
+  - umpire
+  - caliper
+  - faiss
+  - redis-plus-plus
+  - amqp-cpp +tcp
```

Comment:
@koparasy Those can be removed too?

Reply:
redis++ for now yes, amqp needs to be there when we enable RMQ via build options.